src/pet.js

```diff
@@ -0,0 +1,110 @@
+class Pet {    
+    constructor(name) {
+        this.name = name || "An unloved pet";
+        this.age = 0;
+        this.hunger = 0;
+        this.fitness = 10;
+        this.poop = 0;
+        this.children = [];
+    }
+
+    get isAlive() {
+        const MAX_AGE = 30;
+        const MAX_HUNGER = 10;
+        const MIN_FITNESS = 0;
+
+        return this.age < MAX_AGE && this.hunger < MAX_HUNGER && this.fitness > MIN_FITNESS;
+    }
+
+    get deadMessage() {
+        const DEAD_MESSAGE = 'Your pet is no longer alive :(';
+
+        return DEAD_MESSAGE;
```

Comment:
you could also return the string directly, instead of assigning it to a variable.